### PR TITLE
bosses invited again

### PR DESCRIPTION
**Pull request by mandresm**
I think the bosses will have a lot of fun in the party

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,6 @@ Nikolay's latest fancy plots
 
 ## Blacklist (list)
 
-Thomas J.
```
Comment (antoniofis10000) [inline README.md:58]: And Antja?

Comment (clhinrichs) [inline README.md:58]: okay

File: README.md [hunk 2]
```diff
@@ -55,8 +55,6 @@ Nikolay's latest fancy plots
...
-Gerrit L.
```
Comment (clhinrichs) [inline README.md:59]: okay

**Review by antoniofis10000 — COMMENTED** [review]
Check

**Comment by antoniofis10000** [comment]
@mandresm #3 Will they bring something??

File: README.md (hunk 2 above)
Comment (antoniofis10000) [inline README.md:59]: Nop

**Review by antoniofis10000 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mandresm) [reply README.md:59]: I don't agree with this

Comment (mandresm) [reply README.md:59]: I won't do it

**Review by antoniofis10000 — APPROVED** [review]

**Pull request merged**